### DISCUSSION TITLE
Rendering issue on LinuxFramebuffer SingleView

### DISCUSSION THREAD

**Discussion by dedounet** [AvaloniaUI/Avalonia#12944] [Q&A]
Hello everyone, I'm facing a weird issue on the rendering of a popup error warning. 
### Context
Avalonia 0.10.19
.Net 7
Dev machine: Windows 10
Target machine: Debian Lite 

We have created a popup for signaling an error on input control.
When the application is open on a Windows desktop, there are no placement issues. 
![Ok Windows](https://github.com/AvaloniaUI/Avalonia/assets/7623945/5d2ebd5a-f902-47cb-babb-0431d9a4af19)

However, when the application is running on Linux in single view, the placement is not good for the first error occurrence.
![Error Linux FB](https://github.com/AvaloniaUI/Avalonia/assets/7623945/7d275ebb-6555-45cd-9460-e61f8594a656)

Then the placement is good for others control errors.  
![Ok Linux FB](https://github.com/AvaloniaUI/Avalonia/assets/7623945/6c199695-9848-40de-9efe-6e52bfe612ea)

## What I tried 
To resolve this problem, I tried to fix the placement with the variable “PlacementTarget” but that not works.  
I also placed the popup in others positions with “PlacementMode”, but the result was the same.  
This popup is placed in the “ContentTemplate” of a “ContentControl”,   
I also inserted it in the “Content”variable, but this didn’t solve my problem.  

 ```
<Application.Styles> 
    <FluentTheme Mode="Light"/> 
    <Style Selector="DataValidationErrors"> 
      <Setter Property="Template"> 
        <ControlTemplate> 
          <DockPanel LastChildFill="True"> 
            <ContentControl Name="ContentPopUp" DockPanel.Dock="Right" 
                            ContentTemplate="{TemplateBinding ErrorTemplate}" 
                            DataContext="{TemplateBinding Owner}" 
                            Content="{Binding (DataValidationErrors.Errors)}" 
                            IsVisible="{Binding (DataValidationErrors.HasErrors)}"/> 
            <ContentPresenter Name="PART_ContentPresenter" 
                              Background="{TemplateBinding Background}" 
                              BorderBrush="{TemplateBinding BorderBrush}" 
                              BorderThickness="{TemplateBinding BorderThickness}" 
                              CornerRadius="{TemplateBinding CornerRadius}" 
                              ContentTemplate="{TemplateBinding ContentTemplate}" 
                              Content="{TemplateBinding Content}" 
                              Padding="{TemplateBinding Padding}"/> 
          </DockPanel> 
        </ControlTemplate> 
      </Setter> 
      <Setter Property="ErrorTemplate"> 
        <DataTemplate> 
            <Popup IsOpen="{Binding, Converter={ext:ToOpenConverter}}"> 
              <Label Margin="10 0 0 0" Background="Red" BorderBrush="Red"> 
                <ItemsControl Items="{Binding}"/> 
              </Label> 
            </Popup>           
        </DataTemplate> 
      </Setter> 
    </Style> 
  </Application.Styles> 
</Application> 
```

**Comment by timunie** [comment]
can you double-check in a newer Avalonia version? at minimum please update to latest 0.10 release. If possible, please also try 11.0.4 in a minimal sample.

**Reply by dedounet** [reply]
I'm currently using 0.10.19, i'll try by updating to the latest
